mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -113,6 +109,7 @@
 
     public int clientId;
     public List<DataTable> clientDataTables = new ArrayList<>();
+    public String sb;
```

Comment:
ambiguous name, this can be urlStringBuilder, @ishan1604 what you think about that.
I think rest of thing looks good, Please give final look to this PR.